File: Linear_Search/Linear_search.cpp
#include <iostream>
using namespace std;
int main() {
    int n,s;
    cin>>n;
    int arr[n];
    for(int i=0;i<n;i++){
        cin>>arr[i];
    }
    cin>>s;
    for(int i=0;i<n;i++){
        if(s==arr[i]){
            cout<<s<<" Found at index "<<i;
            return 0;
        }
    }
    cout<<"Not Found";
    return 0;
}

File: LinearSearchAlgo/Linearsearch.py
n=int(input())
l=[]
for i in range(n):
    i=int(input())
    l.append(i)
s=int(input())
for i in range(len(l)):
    if(l[i]==s):
        print(s,"Element found at index",i);
        break
else:
    print("Not Found")
    